static/src/javascripts/projects/common/utils/closest.js

```diff
@@ -0,0 +1,18 @@
+define(function () {
+    /**
+     * Adds a wrapper around Element.closest()
+     */
+    var matches = 'matches' in Element.prototype ? 'matches' : 'msMatchesSelector';
```

Comment:
I based this on the assumption that `Element.matches` exists for all browsers that are running commercial, except IE9 which uses `msMatchesSelector`. I may need to add the `webkit` prefix too [for older androids](http://caniuse.com/#feat=matchesselector) though. @ScottPainterGNM do you agree?